packages/ui-components/tsup.config.claim.ts

```diff
@@ -1,8 +1,14 @@
 import { defineConfig } from "tsup"
+import { builtinModules } from "module"
+
+const nodeBuiltins = [
+  ...builtinModules,
+  ...builtinModules.map((m) => `node:${m}`)
+]
 
 export default defineConfig({
   entry: ["src/index.ts"],
-  format: ["esm", "iife"],
+  format: ["esm"], // Only build ESM format since IIFE has issues with external dependencies
```

Comment:
iife is required for web components

Reply:
It worked perfectly fine before your changes
@L03TJ3  please chime in

Reply:
1. Once something like this from your pov breaks the suggested approach is to reach out and ask if we maybe know about a bug or not. The answer would have been: this worked fine before, your changes should not have affected the behavior of the UI components.

2. you chose to use dynamic import of farcaster-sdk? why, I don't see any need for this caching/lazy-loading of the sdk

3. its the dynamic import that is causing issues building the ui-component. adding @farcaster/miniapp-sdk as dependency to citizen-sdk, removing the dynamic/lazy-load import and replace with module import, will have both citizen-sdk and ui-components build without any changes to configuration necessary